plugin/hover.py

```diff
@@ -67,7 +67,7 @@ def run(self, edit: sublime.Edit, point: Optional[int] = None, event: Optional[d
             return
         self._base_dir = windows.lookup(window).get_project_path(self.view.file_name() or "")
         self._hover = None  # type: Optional[Any]
-        self._actions_by_config = {}  # type: Dict[str, List[CodeActionOrCommand]]
+        self._actions_by_config = {}  # type: Dict[str, List[CodeAction]]
```

Comment:
The type is `Dict[str, List[Union[Command, CodeAction]]]` I think.

Reply:
Fixed.